src/isIsogram.test.js

```diff
@@ -8,12 +8,25 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
-
+    expect(typeof isIsogram('1')).toBe('boolean');
```

Comment:
Input can contain only letters, so better choose some words for the input in this test